package.json

```diff
@@ -21,15 +22,15 @@
   "homepage": "https://cmsgov.github.io/qpp-submissions-docs",
   "devDependencies": {
     "@cmsgov/design-system-core": "^1.0.0-alpha.4",
-    "eslint": "^3.19.0",
-    "gh-pages": "0.12.0",
+    "eslint": "^4.3.0",
```

Comment:
Not a huge risk for something like docs but given the security prezo, should we refrain from wildcarding our dependencies?